src/main.cpp

```diff
@@ -638,8 +639,8 @@ int alphabeta(Position &pos,
         if (i == !(no_move == tt_move)) {
             for (int j = 0; j < num_moves; ++j) {
                 const int capture = piece_on(pos, moves[j].to);
-                if (capture != None) {
-                    move_scores[j] = (capture + 1) * (1LL << 54);
+                if (capture) {
+                    move_scores[j] = capture * (1LL << 54);
```

Comment:
If the type of `capture` is changed, can this line become `move_scores[j] = capture << 54;`?
Using `u64` saves 1 byte but is not exactly great code.

Reply:
Sure?